src/views/GovernanceReviewTeam/Notes/index.tsx

```diff
@@ -106,35 +113,37 @@ const Notes = () => {
         content: values.note
       })
     );
-    notesRefetch();
     await resetForm();
+    if (refetch) {
+      // I was having an issue where sometimes refetch would be undefined.
+      // This may be related to a bug in Apollo Client, so the check here is
+      // to prevent that from breaking the app should it occur.
+      refetch();
+    }
   };
 
-  useEffect(() => {
-    dispatch(fetchActions(systemId));
-  }, [dispatch, systemId]);
-
   const initialValues = {
     note: ''
   };
 
   const notesByTimestamp =
-    notesData?.systemIntake?.notes.map(note => {
+    data?.systemIntake?.notes.map(note => {
       return {
         createdAt: note.createdAt,
         element: <NoteListItem note={note} key={note.id} />
       };
     }) || [];
 
-  const actionsByTimestamp = actions.map((action: Action) => {
-    return {
-      createdAt: action.createdAt,
-      element: <ActionListItem action={action} key={action.id} />
-    };
-  });
+  const actionsByTimestamp =
+    data?.systemIntake?.actions.map(action => {
+      return {
+        createdAt: action.createdAt,
+        element: <ActionListItem action={action} key={action.id} />
+      };
+    }) || [];
 
   const interleavedList = [...notesByTimestamp, ...actionsByTimestamp]
-    .sort()
+    .sort((a, b) => (a.createdAt > b.createdAt ? 1 : -1))
```

Comment:
I removed this in #912 and broke sorting since we need to sort based on the `createdAt` property of the elements and not the elements in the collection themselves.